lib/deas/router.rb

```diff
@@ -140,11 +140,12 @@ def apply_definitions!
     def validate_trailing_slashes!
       if self.trailing_slashes == REMOVE_TRAILING_SLASHES
         paths = []
-        all_missing = self.routes.inject(true) do |result, route|
-          paths << route.path if route.path[-1..-1] == SLASH
-          result && route.path[-1..-1] != SLASH
+        has_trailing = self.routes.inject(false) do |result, route|
+          route_has = route.path[-1..-1] == SLASH && route.path != SLASH
```

Comment:
@kellyredding - This is a little pedantic but you might use `route.path[-1, 1]` instead to get the last character. I feel like it's a little easier to read and maybe more efficient in that it doesn't have to build a range. Not a big deal if you prefer it the way it is.

Reply:
@jcredding nah, this is a good call.  thanks.